src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { render, RenderResult, screen } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import type { JiraIssue } from 'services/cf'
+import JiraIssueList, { JiraIssueListProps } from '../JiraIssueList'
+
+const renderComponent = (props?: Partial<JiraIssueListProps>): RenderResult => {
+  const jiraIssues: JiraIssue[] = [
+    {
+      issueKey: 'ISSUE_1',
+      issueURL: 'https://jira.com/ISSUE_1'
+    },
+    {
+      issueKey: 'ISSUE_2',
+      issueURL: 'https://jira.com/ISSUE_2'
+    }
+  ]
+
+  return render(
+    <TestWrapper
+      path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/feature-flags"
+      pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
+    >
+      <JiraIssueList jiraIssues={jiraIssues} {...props} />
+    </TestWrapper>
+  )
+}
+
+describe('JiraIssueList', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
```

Comment:
Reset is fine in NGUI, but when we plop this in FF-UI it'll kill the `TestWrapper` (the test wrapper mocks things in FF-UI and is all hard-coded in NGUI). In this specific case, you're not mocking anything, so you can drop it altogether, but in general favour `clearAllMocks` over `resetAllMocks`
```suggestion
    jest.clearAllMocks()
```

Reply:
removed